### PR TITLE
Insight logs docs

### PR DESCRIPTION
**Pull request by Konstantinov-Innokentii**
# What this PR does
Brings insight logs docs and polish format of insight logs

### CODE REVIEW

File: docs/sources/insight-logs/_index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Grafana OnCall insight logs
+
+Grafana OnCall insights logs represents certain activities, such as:
+
+- A user creates, updates or deletes resource.
+- A Maintenance mode is started or finished for an integration.
+- A user configure ChatOps integration.
```
Comment (Konstantinov-Innokentii) [inline docs/sources/insight-logs/_index.md:7]: I don't like this description, it is too wide.  Actually chatops insight logs created every time when user link/unlink messaging backend, link/unlink channel or install slack, but it seems to long to me

File: docs/sources/insight-logs/_index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+Logs of maintenance insights contain the following fields, where the fields followed by * are always available, and the others depend on the logged event:
+
+| Field Name          |                               Description                                |
+|---------------------|:------------------------------------------------------------------------:|
+| `action_name`*      |   Name of the maintenance action, which can be `started` or `finised`.   |
+| `action_type`*      | Insight Log type. For Maintenance Insight logs it will be `maintenance`. |
+| `author`            |                  Username of user who performed action.                  |
+| `author_id`         |             Grafana OnCall ID of user who performed action.              |
+| `maintenance_mode`* |      Type of the maintenance, which can be `maintenance` or `debug`      |
+| `resource_id`*      |                  ID of integration that was maintained.                  |
```
Comment (Konstantinov-Innokentii) [inline docs/sources/insight-logs/_index.md:71]: 'was maintained' sounds weird, I tried to say "maintenance was started or finished", but shorter

File: docs/sources/insight-logs/_index.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Grafana OnCall insight logs
+
```
[the remark this answers is not in the document]
Comment (Matvey-Kuk) [reply docs/sources/insight-logs/_index.md:2]: Grafana OnCall insight logs are available in Grafana Cloud only. We're in process of rolling Insight Logs to all customers, if you don't see insight logs in your Grafana cloud stack, please reach out to support.

[the remark this answers is not in the document]
Comment (Matvey-Kuk) [reply docs/sources/insight-logs/_index.md:2]: Please rename folder to /docs/sources/insights to leave space for metrics docs here.

**Review by Matvey-Kuk — APPROVED** [review]

File: docs/sources/insight-logs/_index.md (hunk 2 above)
Comment (Matvey-Kuk) [reply docs/sources/insight-logs/_index.md:71]: - ID of target integration.
- ID of integration under maintenance.
- ID of maintained integration.

File: docs/sources/insight-logs/_index.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+{instance_type="oncall"} | logfmt | __error__=`` | action_type = `chat_ops`
+```
+
+### Format
+
+Logs of chatops insight logs contain the following fields, where the fields followed by * are always available, and the others depend on the logged event:
+
+| Field Name       |                                   Description                                    |
+|------------------|:--------------------------------------------------------------------------------:|
+| `action_name`*   |                            Name of the chatops action                            |
```
[the remark this answers is not in the document]
Comment (Matvey-Kuk) [reply docs/sources/insight-logs/_index.md:92]: Please add types, position on your taste.

File: docs/sources/insight-logs/_index.md (hunk 2 above)
Comment (Matvey-Kuk) [reply docs/sources/insight-logs/_index.md:71]: Please add query examples

File: docs/sources/insights/_index.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+This configuration is done for you in Grafana Cloud with [Usage Insights Loki data source](https://grafana.com/docs/grafana-cloud/billing-and-usage/usage-insights/#usage-insights-loki-data-source).
+You can use this query to retrieve all logs related to your OnCall instance.
+
+```logql
+{instance_type="oncall"} | logfmt | __error__=``
+```
+
+## Resource insight logs
+
+Logs are created each time user modifies any resource in Grafana OnCall.
```
Comment (joeyorlando) [inline docs/sources/insights/_index.md:22]:
```suggestion
Logs are created each time a user modifies any resource in Grafana OnCall.
```

File: docs/sources/insights/_index.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
+# Grafana OnCall insight logs
+
+> **Note:** Grafana OnCall insight logs are available in Grafana Cloud only.
+We're in process of rolling Insight Logs to all customers,
+if you don't see insight logs in your Grafana cloud stack, please reach out to support.
+
+Grafana OnCall insights logs represents certain activities, such as:
```
Comment (joeyorlando) [inline docs/sources/insights/_index.md:7]:
```suggestion
Grafana OnCall Insights Logs represents certain activities, such as when:
```

File: docs/sources/insights/_index.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+- A user creates, updates or deletes resource.
```
Comment (joeyorlando) [inline docs/sources/insights/_index.md:9]:
```suggestion
- A user creates, updates, or deletes a resource.
```

File: docs/sources/insights/_index.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+- A Maintenance mode is started or finished for an integration.
+- A user configure ChatOps integration.
```
Comment (joeyorlando) [inline docs/sources/insights/_index.md:11]:
```suggestion
- A user configures a ChatOps integration.
```

File: docs/sources/insights/_index.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+| `channel_name`   |                Name of the channel linked to chatops integration                 |
+| `prev_channel`   |                      Name of team to which resource belongs                      |
+| `new_channel`    |               Grafana OnCall ID of team to which resource belongs                |
+
+chatops action names: `workspace_connected`, `workspace_disconnected`, `channel_connected`, `channel_disconnected`, `user_linked`, `used_unlinked`, `default_channel_changed`.
+
+## Examples
+
+Here is some examples of practical queries to Grafana OnCall insight logs.
+LogQL is used to retrieve them, If you are not familiar with LogQL check this [documentation](https://grafana.com/docs/loki/latest/logql/).
```
Comment (joeyorlando) [inline docs/sources/insights/_index.md:116]:
```suggestion
LogQL is used to retrieve them. If you are not familiar with LogQL, check out this [documentation](https://grafana.com/docs/loki/latest/logql/).
```

**Review by joeyorlando — APPROVED** [review]
LGTM! some small grammatical things

**Pull request merged**